packages/app-desktop/gui/ConfigScreen/configScreenUtils.ts

```
@@ -0,0 +1,16 @@
+import { AppType, SettingItem } from '@joplin/lib/models/Setting';
+
```

ESLint

[error] Expected linebreaks to be 'LF' but found 'CRLF'.

(linebreak-style)